src/WebJobs.Script/Eventing/File/FileEventSources.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Eventing
+{
+    public static class FileEventSources
```

Comment:
Wondering if you don't instead just want a general ScriptEventSources class for all the various sources, not file specific?